.travis.yml

```diff
@@ -146,14 +118,3 @@ jobs:
                 - docker-compose exec php bin/console sylius:fixtures:load --no-interaction
 
                 - curl http://localhost/
-
-            before_deploy:
```

Comment:
It has never been deployed before and I don't think we can find the time (if anyone would like to give it a try, let me know).